src/main/webapp/app/tasks/task-list/task-list.component.scss

```diff
@@ -1,14 +1,35 @@
+$brand_color: #9bbf29;
```

Comment:
Nice :)

Reply:
Thanks 😄 . It is a good practice to make any updates when we identify an improvement while we are working on the same file.